atomicapp/nulecule/base.py

```diff
@@ -299,6 +299,11 @@ def get_artifact_paths_for_provider(self, provider_key):
                 path = Utils.sanitizePath(artifact)
                 path = os.path.join(self.basepath, path) \
                     if path[0] != '/' else path
+            elif isinstance(artifact, dict) and artifact.get('inherit') and \
+                    isinstance(artifact.get('inherit'), list):
+                inherited_provider_key = artifact.get('inherit')[0]
```

Comment:
So you check if it's a `list` and then blindly use first item? What about the rest?:-) (i.e. I am missing some `for` loop here:) )